getsolus/packages
# vice: Update to 3.8

**Summary**

- Full changelog can be found [here](https://vice-emu.sourceforge.io/NEWS)
- Added appstream metadata (Part of getsolus/packages#1389)
- Added desktop files

**Test Plan**

- Run all binaries installed
- Verify metadata with `appstream-builder --packages-dir=. --include-failed -v`

**Checklist**
